dji_interface/src/dji_interface.cpp

```diff
@@ -171,6 +171,7 @@ void DJIInterface::setPublishers()
   imu_pub_ = nh_.advertise<sensor_msgs::Imu>(mav_msgs::default_topics::IMU, 1);
   rc_pub_ = nh_.advertise<sensor_msgs::Joy>(mav_msgs::default_topics::RC, 1);
   status_pub_ = nh_.advertise<mav_msgs::Status>(mav_msgs::default_topics::STATUS, 1);
+  gps_pub_ = nh_.advertise<sensor_msgs::NavSatFix>("gps", 1); //Hard-coded topic name, change this later.
```

Comment:
use `mav_msgs::default_topics::GPS`. It is already defined here https://github.com/ethz-asl/mav_comm/blob/f8fafd45e34c6c08c08f00175742c7d8dc98c7cf/mav_msgs/include/mav_msgs/default_topics.h#L30

Reply:
That's what I was looking for.
LGTM.